### PR TITLE
fix page style

### PR DESCRIPTION
**Pull request by turbo-stump**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,513 +1,512 @@
...
+  const currentScoreImageElement = document.createElement("img");
+  const currentScoreElementChild = Array.prototype.slice.call(
+    currentScoreElement.childNodes
+  );
+
+  currentScoreElementChild.forEach((child) =>
+    currentScoreElement.removeChild(child)
+  );
+
+  const digits = score.toString().split("");
```
Comment (stuffy-idler) [inline js/main.js:293]: nit: can be written in FP-style as:
```typescript
score.toString().split("").forEach(digit => {
  currentScoreImageElement.src = `../img/assets/font_small_${digit}.png`;
  currentScoreImageElement.alt = `${digit}`;
  currentScoreElement.append(currentScoreImageElement);
});
```

File: js/main.js [hunk 2]
```diff
@@ -1,513 +1,512 @@
...
+}
 
-    //have they tried to escape through the ceiling? :o
-    var ceiling = $("#ceiling");
-    if (boxtop <= ceiling.offset().top + ceiling.height()) position = 0;
+function setMedal() {
+  const medalElement = document.querySelector("#medal");
+  const medalImageElement = document.createElement("img");
+  const medalElementChild = Array.prototype.slice.call(medalElement.childNodes);
+  medalElementChild.forEach((child) => medalElement.removeChild(child));
```
Comment (stuffy-idler) [inline js/main.js:322]: nit: can be rewritten as `medalElementChild.forEach(medalElement.removeChild)`

File: index.html [hunk 3]
```diff
@@ -110,18 +113,15 @@ <h1 class="union__logo-text">
...
+    document.querySelector(".game-btn").addEventListener("click", () => {
+      gtag("event", "game_click", {
+        event_label: "Start game",
+      });
+    });
   </script>
 </body>
 
-</html>
+</html>
```
Comment (stuffy-idler) [inline index.html:127]: Missing EOF

**Review by stuffy-idler — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (turbo-stump) [reply js/main.js:293]: done

File: js/main.js [hunk 4]
```diff
@@ -1,513 +1,512 @@
...
+function setHighScore() {
+  const highScoreElement = document.querySelector("#highscore");
+  const highScoreImageElement = document.createElement("img");
+  const highScoreElementChild = Array.prototype.slice.call(
+    highScoreElement.childNodes
+  );
+
+  highScoreElementChild.forEach((child) => highScoreElement.removeChild(child));
+
+  const digits = score.toString().split("");
```
[the remark this answers is not in the document]
Comment (turbo-stump) [reply js/main.js:310]: done

File: js/main.js (hunk 2 above)
Comment (turbo-stump) [reply js/main.js:322]: done

File: js/main.js [hunk 5]
```diff
@@ -1,513 +1,512 @@
...
-
-    //if we're in debug mode, draw the bounding box
-    if (debugmode) {
-      var boundingbox = $("#playerbox");
-      boundingbox.css("left", boxleft);
-      boundingbox.css("top", boxtop);
-      boundingbox.css("height", boxheight);
-      boundingbox.css("width", boxwidth);
-    }
+  const digits = score.toString().split("");
```
[the remark this answers is not in the document]
Comment (turbo-stump) [reply js/main.js:272]: done

File: index.html (hunk 3 above)
Comment (turbo-stump) [reply index.html:127]: done

File: js/main.js [hunk 6]
```diff
@@ -1,513 +1,512 @@
...
+}
+
+function setSmallScore() {
+  const currentScoreElement = document.querySelector("#currentscore");
+  const currentScoreImageElement = document.createElement("img");
+  const currentScoreElementChild = Array.prototype.slice.call(
+    currentScoreElement.childNodes
+  );
+
+  currentScoreElementChild.forEach((child) =>
```
[the remark this answers is not in the document]
Comment (stuffy-idler) [reply js/main.js:290]: Leads to Illegal invocation

**Review by stuffy-idler — APPROVED** [review]

**Review by stupid-boar — APPROVED** [review]

**Pull request merged**